specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,552 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-01-01",
+    "title": "Marketplace RP Service"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
```

Comment:
>paths [](start = 3, length = 5)

which of these resources are ARM tracked resources ? (routing type=manifest. ?
For all those, PATCH must be defined, with support for patching at least 'tags'.

Could you please create manifest ARM manifest PR as well ?
(repo: [https://armmanifest.visualstudio.com/One/_git/Manifest](https://armmanifest.visualstudio.com/One/_git/Manifest))

Reply:
it will be tenant proxy no arm track